package.json

```diff
@@ -20,6 +20,7 @@
     "framer-motion": "^4.1.17",
     "geolib": "^3.3.3",
     "next": "^11.1.3",
+    "node-fetch": "^3.2.0",
```

Comment:
I think we can remove the this dependency from the `package.json` as long as it's bumped in the `yarn.lock`